apps/aefate/src/aefa_engine_state.erl

```diff
@@ -6,7 +6,7 @@
 %%%-------------------------------------------------------------------
 -module(aefa_engine_state).
 
--export([ new/6
+-export([ new/7
```

Comment:
I think we need an access function as well ;-)

Reply:
right!